### ISSUE TITLE
Caps Lock Activated Indication in message box that requires a password 

### ISSUE THREAD

**Issue by danielschickmunit** [microsoft/vscode#148609]
Create a "Caps Lock Activated" - Indication when entering a password for git or a ssh connection
(this window here)
![image](https://user-images.githubusercontent.com/104583173/166413056-cee870b1-3296-415f-8e8b-f089dd76b287.png)

**Comment by lszomoru** [comment]
@TylerLeonhardt, is this something that should be implemented in the quick input?

**Comment by TylerLeonhardt** [comment]
Yep. I think all password quick pick's should have this feature.
